BootloaderCorePkg/Library/SmbiosInitLib/SmbiosInitLib.c

```diff
@@ -18,6 +18,32 @@
 
 VOID      *mType127Ptr            =   NULL;
 
+//
+// Add more platform specific strings
+// in the following format
+//
+SMBIOS_TYPE_STRINGS  DefaultSmbiosStrings[] = {

```

Comment:
Use  mDefaultSmbiosStrings to follow naming convention for global variable.
Also add CONST.